code/modules/mining/abandoned_crates.dm

```diff
@@ -187,6 +187,17 @@
 		return
 	attack_hand(user)
 
+/obj/structure/closet/crate/secure/loot/bullet_act(obj/item/projectile/P)
+	if(prob(2)) //parent has 1000 health, 20 damage weapon has 36% chance of getting the loot.
+		boom()
```

Comment:
I can put the same "if burn or brute" check that depleting the health does if that's preferred.